Wikipedia/Code/WMFNearbyContentSource.m

```diff
@@ -59,20 +63,35 @@ - (WMFLocationSearchFetcher *)locationSearchFetcher {
 
 - (void)startUpdating {
     self.isFetchingInitialLocation = NO;
-    [self.currentLocationManager startMonitoringLocation];
+    if ([WMFLocationManager isAuthorized]) {
+        [self.currentLocationManager startMonitoringLocation];
+    }
 }
 
 - (void)stopUpdating {
-    [self.currentLocationManager stopMonitoringLocation];
+    if ([WMFLocationManager isAuthorized]) {
```

Comment:
Is this scenario possible:
 - Location services are authorized
 - startUpdating is called
 - Location services are de-authorized (user goes into Settings and disables or something)
 - stopUpdating is called, but stopMonitoringLocation isn't

Basically, is there harm in always calling `[self.currentLocationManager stopMonitoringLocation]` regardless or authorization?